### ISSUE TITLE
Accuracy values during multi-GPU training

### ISSUE THREAD

**Issue by tboo** [ludwig-ai/ludwig#1302]
Hi,

exploring ludwig 0.4 for the first time after a long ludwig break. love the improvements. Great job, as always.

I came across some strange behavior when training with multiple GPUs using Horovod in an image classification problem. When training on a single GPU (w/o) horovod I get an accuracy <1 that progresses toward 1 during training. So far so good. Now, when I use Horovod this accuracy somewhat scales with the number of GPUs I am using. E.g. it seems to add up the accuracy from each GPU. (With two GPUs the accuracy converges to 2, with three GPU to 3 and so on).

My question is simple: is this a bug or a feature?

Single GPU training:

```ludwig train --dataset annotation_200pxl.csv --config_file model_definition_std_cnn.yaml```

Multi GPU training:

```horovodrun -np 2 ludwig train --dataset annotation_200pxl.csv --config_file model_definition_std_cnn.yaml```

with a configuration file that reads:

```
input_features:
    -
        name: image_path
        type: image
        encoder: resnet

output_features:
    -
        name: class
        type: category

training:
    reduce_learning_rate_on_plateau: 10
    early_stop: 10
    learning_rate: 0.0004
    batch_size: 6
```

Thanks so much for the feedback

**Comment by connor-mccorm** [comment]
This issue should be resolved in the latest Ludwig version.
